src/components/autocomplete/Autocomplete.vue

```diff
@@ -316,6 +316,10 @@ export default {
                     }, 100)
                 }
             }
+
+            this.$nextTick(() => {
```

Comment:
@daniil4udo Do you have any specific reason for deferring emit to the next tick?

Reply:
It works anyway.